sdks/python/apache_beam/runners/interactive/interactive_environment_test.py

```diff
@@ -180,91 +180,81 @@ def test_cleanup_invoked_when_new_env_replace_not_none_env(
       self, mocked_cleanup):
     ie._interactive_beam_env = None
     ie.new_env()
-    mocked_cleanup.assert_not_called()
     ie.new_env()
-    mocked_cleanup.assert_called_once()
+    mocked_cleanup.assert_called()
```

Comment:
This seems to change the meaning of the test. Do we not want to verify that cleanup is not called when _interactive_beam_env is None?

Reply:
Yes, a little bit. We don't really care about if cleanup is called when _interactive_beam_env is None, because it's a NOOP either way. In this scenario, we are testing when ie.new_env() is called more than once (so that a new env did replace an old env), cleanup is called at least once.